modules/quotas-resources-managed.adoc

```diff
@@ -99,6 +99,9 @@ A pod is in a terminal state if `status.phase in (Failed, Succeeded)` is true.
 |`persistentvolumeclaims`
 |The total number of persistent volume claims that can exist in the project.
 
+|`count/replicasets.apps`
+|The total number of ReplicaSets that can exist in the project.
```

Comment:
@dbaker-rh is the object type `ReplicaSet`? If so, let's phrase it this way: 
```suggestion
|The total number of `ReplicaSet` objects that can exist in the project.
```
This suggestion is based on the following guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

Reply:
It is, but my text stays in line with the phrasing of the items in the list above (with the exception of the line immediately above).

I have no problem with the rephrase, but if we're following that guideline it might be nice to apply it more widely to this list.

Reply:
You're right that they should also follow the same guideline. You can expand the scope of this PR or open an issue to address that need. When I conduct peer reviews, I confine my comments to the changes at hand.

Reply:
I'm happy with either text, then.  Mine, to stay consistent.  Or yours, to pre-empt future changes to come.

Reply:
I recommend "`ReplicaSet` objects" because  "ReplicaSets" leaves some ambiguity about the exact name of the object.